src/Microsoft.VisualStudio.Editors/PropPages/AdvBuildSettingsPropPage.vb

```diff
@@ -352,6 +347,13 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                 End If
             Next
         End Sub
+
+        Private Sub LnkLabel_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles lnkLabel.LinkClicked
+            TelemetryLogger.LogAdvBuildSettingsPropPageEvent(TelemetryLogger.AdvBuildSettingsPropPageEvent.LangVersionMoreInfoLinkClicked)
```

Comment:
What business decision will we make in regards to the user clicking this? If we are making a business decision on this, might this be answered by looking at the aka link numbers, or the page views?

Reply:
(Note: I didn't request this specific point, but I'll attempt to rationalize it below)

Both page views and aka link numbers could be skewed because you can access the information outside of VS. I don't think we'd make a business decision about this since we can't for sure say if clicking on this link indicates dissatisfaction, though, so if we can't find a business decision for this specific data point I'm not going to die on a hill to keep it.